### ISSUE TITLE
Text replacements and UTF-8 presets (like "->" to "➔")

### ISSUE THREAD

**Issue by Flockcode** [AuraDevelopmentTeam/BungeeChat2#112]
Very nice and useful plugin! We use global chat and it is amazing. I would still like to have some features. For example that emotes like <3 - would be converted into a real Hart. I would love it here if everything was configurable. The second thing I would like is to have player names highlighted. If the player's name is yours, I'd love the plugin to play a sound (I'm not sure if that's possible). If you have further questions, you can contact me at Discord Hohi#1865. 

I know that my English is not the best.
Best wishes 
Hohi

**Comment by BrainStone** [comment]
The second part is already covered in feature request #52.

Though I'm not sure if emojis are possible in Minecraft.  
And I'm even less sure what you'd want to be configurable there.

**Comment by Flockcode** [comment]
Thanks for you're response, I saw unicode characters on some server. I don't really want emojis. Just that the Plugin convert -> to ➔ for example. I know that it's possible to replace text. There are some characters that are supported in Minecraft.  

I'm very sorry, that I used the wrong title and posted the same thing on spigot. I know many developers, that never respond. Now I know, that you aren't like that.

If you have some more Questions, please aks me :D
- Hohi

**Comment by Flockcode** [comment]
Wenn ich gewusst hätte, dass Du deutscher bist, hätte ich es auch auf Deutsch schreiben können xD
Liebe Grüsse aus der Schweiz

**Comment by BrainStone** [comment]
Please keep discussions on GitHub in English. I'd like to avoid having duplicates simply because the new OP didn't understand the language of the other issue. ;)

And now I understand.

**Comment by BrainStone** [comment]
I have a little work for you to do:

Could you please collect a list of replacements you think would be good defaults? (They will be turned off. But as a "starter" kit of sorts).

**Comment by jewome62** [comment]
Maybe pick some into LibreOffice Calc into AutoCorrect ?
I see
```
![image](https://user-images.githubusercontent.com/472429/80811072-74a4d880-8bc5-11ea-8584-a21a789599d7.png)

**Comment by AwesomestCode** [comment]
@BrainStone After looking at the source code a bit, I found two places that jumped out to me as candidates for a place to implement. Which one is better: at `preProcessMessage()` or `sendToMatchingPlayers()`? I believe both are used by all chat listeners at least indirectly.

**Comment by BrainStone** [comment]
That's stuff for a preprocessor. Could actually also be done with an addon, as they can add preprocessors.

**Comment by AwesomestCode** [comment]
So far, I have a hardcoded version (meaning that you can't change the replacements) done.Now I need to get it to work with the config...

**Comment by BrainStone** [comment]
Alright we need to collect a decent list of preset replacements.

**Comment by BrainStone** [comment]
Anyone...?  
I'm not the someone that asked for that feature. The code is ready all we need are some decent defaults to put in the config and we are good. I have a released planned soon and I'd like to include that code.

**Comment by AwesomestCode** [comment]
Maybe mention @Flockcode?

**Comment by Flockcode** [comment]
Hey, sorry - I was AFK for nearly a month. I like your list @AwesomestCode

**Comment by BrainStone** [comment]
I'm still waiting on a decent default list of replacements.

**Comment by BrainStone** [comment]
Why did you close the issue?

**Comment by Georg-code** [comment]
Okey, I'll do a list:
:( = ☹
:) = ☺
<3 =  ♥ 
-> = →
<- = ←
? = ツ
? = ♫
? = ☢

// ? = Usefull, but don't know a code, maby somthing like Github and Discord use -> :note: = ♫

I hope this helps. If anyone wants to add more, here is a list. https://hypixel.net/threads/ツsymbols☻.787483/

**Comment by BrainStone** [comment]
Thanks though I still think that list is very short. Especially a lot of smileys are missing
